src/domains/messages-common/03_database.tf

```
@@ -131,4 +131,86 @@ resource "azurerm_key_vault_secret" "mongodb_connection_string_reminder" {
   value        = module.cosmosdb_account_mongodb_reminder.connection_strings[0]
   content_type = "full connection string"
   key_vault_id = module.key_vault.id
+}
+
+#
+# Reminder PostgreSQL
+#
+// db admin user credentials
+data "azurerm_key_vault_secret" "reminder_postgresql_db_server_adm_username" {
+  name         = "${module.reminder_postgresql_db_server.name}-DB-ADM-USERNAME"
+  key_vault_id = data.azurerm_key_vault.common.id
+}
+data "azurerm_key_vault_secret" "reminder_postgresql_db_server_adm_password" {
+  name         = "${module.reminder_postgresql_db_server.name}-DB-ADM-PASSWORD"
+  key_vault_id = data.azurerm_key_vault.common.id
+}
```

```suggestion
data "azurerm_key_vault_secret" "reminder_postgresql_db_server_adm_username" {
  name         = "${local.product}-${var.domain}-DB-ADM-USERNAME"
  key_vault_id = data.azurerm_key_vault.key_vault.id
}

data "azurerm_key_vault_secret" "reminder_postgresql_db_server_adm_password" {
  name         = "${local.product}-${var.domain}-DB-ADM-PASSWORD"
  key_vault_id = data.azurerm_key_vault.key_vault.id
}
```
you cannot use module postgresql output because you create a circular dependency (postgresql module needs admin username and password from kv)